### ISSUE TITLE
New compilation failures on Linux systems

### ISSUE THREAD

**Issue by eserte** [plicease/Alien-Libarchive2#5]
My Linux smokers (Debian wheezy and jessie, all perl versions) started to report compilation failures:
```
libarchive/archive_write_add_filter_xz.c:227:2: error: unknown type name 'lzma_mt'
  lzma_mt mt_options;
  ^
libarchive/archive_write_add_filter_xz.c:237:14: error: request for member 'threads' in something not a structure or union
    mt_options.threads = data->threads;
              ^
libarchive/archive_write_add_filter_xz.c:238:14: error: request for member 'timeout' in something not a structure or union
    mt_options.timeout = 300;
              ^
libarchive/archive_write_add_filter_xz.c:239:14: error: request for member 'filters' in something not a structure or union
    mt_options.filters = data->lzmafilters;
              ^
libarchive/archive_write_add_filter_xz.c:240:14: error: request for member 'check' in something not a structure or union
    mt_options.check = LZMA_CHECK_CRC64;
              ^
```
This does not happen on my FreeBSD smokers.

**Comment by plicease** [comment]
I get this too now.  Although I didn't before.  I think libarchive is part of the OS on FreeBSD, so I suspect the OS version of libarchive is being used (?) which would skip this part of the build.

**Comment by plicease** [comment]
I think we have an incompatible xz version:
https://github.com/libarchive/libarchive/commit/7c3148ec0a2fed4834c6f9869ff7d2da77ba804c

**Comment by plicease** [comment]
I believe this to be fixed in Alien::Libarchive::Installer 0.15 and Alien::Libarchive 0.28

thanks for the report!
